src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseApprovalService.cs

```diff
@@ -145,7 +145,7 @@ IEmailTemplateService emailTemplateService
                     
                     var userPublicationRoles = await _context.UserPublicationRoles
                         .Include(upr => upr.User)
-                        .Where(upr => upr.PublicationId == release.PublicationId && (upr.Role == PublicationRole.Owner || upr.Role == PublicationRole.ReleaseApprover))
+                        .Where(upr => upr.PublicationId == release.PublicationId && upr.Role == PublicationRole.ReleaseApprover)
```

Comment:
Could move both of the `userReleaseRoles` and `userPublicationRoles` variables lower down to just below
`case ReleaseApprovalStatus.HigherLevelReview:`